src/NServiceBus.RabbitMQ/Receiving/MessagePump.cs

```diff
@@ -165,6 +167,30 @@ void Connection_ConnectionShutdown(object sender, ShutdownEventArgs e)
 
             try
             {
+                // The current thread will be the event raising thread if either:
```

Comment:
Minor grammatical nitpick: Shouldn't this line either be "the event-raising thread" or "the thread raising the event " ?